multisig-wallet/project/contracts/multisig-contract/src/main.sw

```diff
@@ -200,6 +200,7 @@ impl Info for Contract {
 /// Recovers a b256 address from each signature;
 /// it then increments the number of approvals by that address' approval weighting.
 /// Returns the final approval count.
+#[inline(never)]
```

Comment:
hm, what did you encounter which made you add this annotation?

Reply:
I got a compiler error, which was only resolved with this annotation. After reporting it seems that it was caused by some `asm` in a function called by this function.

Reply:
Sounds like a bug, we should remove the annotation in the future